experimental/regression_suite/README.md

```diff
@@ -50,9 +50,92 @@ source /tmp/iree_gh_venv/bin/activate
 
 In the above, `<<RUN_ID>>` is the value in any GitHub action presubmit/postsubmit
 workflow which has built package artifacts. As an example, if looking at a GitHub
-Actions status page on: 
+Actions status page on:
 `https://github.com/openxla/iree/actions/runs/5957351746/job/16159877442`, then
 the run id is the first number in the URL (5957351746).
 
 Running the above will allow you to run `pytest` and you will have tools as built
 at the commit from which the workflow run originated.
+
+## Developing using Visual Studio Code
+
+You can also run and debug this test suite using
+[Visual Studio Code](https://code.visualstudio.com/):
+
+![vscode_test_explorer](docs/vscode_test_explorer_annotated-min.png)
```

Comment:
Rich display isn't loading for me on GitHub's PR review page. See a preview here: https://github.com/ScottTodd/iree/tree/tests-vscode-setup/experimental/regression_suite

(I crunched these .png files with https://compresspng.com/ to keep the repo smaller... maybe that confused it)